readme.md

```diff
@@ -0,0 +1,16 @@
+#Project64  
+![Project64](http://i.imgur.com/8b6Qono.png)
+###A Nintendo 64 Emulator  
+
+###Licence  
+Licenced under GPLv2 (See [Licence.md](https://github.com/project64/project64/blob/master/licence.md))
+
+###Downloading
+You can download the latest Public Stable build [Here](http://www.pj64-emu.com/downloads/project64/binaries/)  
+If you would like a Not-So-Stable build, you can either Build it yourself (See Building) or use the unofficial build [Here](http://hyvart.com/download/project64/project64.application)
```

Comment:
Can you please update the link to the ClickOnce app?
http://hyvart.com/download/project64/project64dev.application

Eventually, the project64.application will route to a stable (release) build, where `project64dev.application` will keep fetching the continuous integration builds.